### PR TITLE
Move to frostfs-api

### PR DESCRIPTION
**Pull request by fyrchik**
Depends on TrueCloudLab/frostfs-crypto#1

Signed-off-by: Evgenii Stratonikov <e.stratonikov@yadro.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -23,5 +23,5 @@ require (
 )
 
 // This version uses broken NeoFS API with incompatible signature
-// definitions. See fix in https://github.com/nspcc-dev/neofs-api/pull/203
+// definitions. See fix in https://github.com/nspcc-dev/frostfs-api/pull/203
 retract v2.12.0
```
Comment (alexvanin) [inline go.mod]: We are dropping old tags, so we can drop this retract as well.

File: go.mod [hunk 2]
```diff
@@ -1,9 +1,9 @@
-module github.com/nspcc-dev/neofs-api-go/v2
+module github.com/TrueCloudLab/frostfs-api-go/v2
 
 go 1.17
 
 require (
-	github.com/nspcc-dev/neofs-crypto v0.4.0
+	github.com/TrueCloudLab/frostfs-crypto v0.5.0
```
Comment (alexvanin) [inline go.mod:6]: Push this tag. All other tags were removed from frostfs-crypto.

**Review by alexvanin — DISMISSED** [review]

File: go.mod (hunk 1 above)
Comment (fyrchik) [reply go.mod]: Fixed

File: go.mod (hunk 2 above)
Comment (fyrchik) [reply go.mod:6]: Fixed

**Review by alexvanin — APPROVED** [review]

**Pull request merged**